translations/zh-cn/README.md

```diff
@@ -0,0 +1,257 @@
+或者导入 ES 模块：
+
+```js
+import quicklink from "dist/quicklink.mjs";
+quicklink();
+```
+
+以上方法适用于多页网站。单页应用可以搭配 router 使用 quicklink：
+
+- 进入新路由地址后，调用 `quicklink()`。
+- 针对特定 DOM 元素/组件调用 `quicklink()`。
+- 调用 `quicklink({urls：[...]})`，传入自定义 URL 集合进行预获取。
+
+## API
+
+`quicklink` 接受带有以下参数的 option 对象（可选）：
+
+- `el`：指定需要预获取的 DOM 元素视区。
+- `urls`：预获取的静态 URL 数组（若此参数非空，则不会检测视区中 `document` 或 DOM 元素的链接）。
+- `timeout`：整型数，为 requestIdleCallback 设置超时。浏览器必须在此之前进行预获取（以毫秒为单位）， 默认取 2 秒。
+- `timeoutFn`：指定超时处理函数。默认为 requestIdleCallback。也可以替换为 [networkIdleCallback](https://github.com/pastelsky/network-idle-callback)（详见 demo）等自定义函数。
+- `priority`：布尔值，指定 fetch 的优先级。默认为 `false`。若配置为 `true` 将会尝试使用 `fetch()` API（而非 rel=prefetch）。
+- `origins`: 静态字符串数组，包含允许进行预获取操作的 URL 主机名。默认为同域请求源，可阻止跨域请求。
+- `ignores`: RegExp（正则表达式），Function（函数）或者 Array（数组），用于进一步确定某 URL 是否可被预获取。会在匹配请求源之后执行。
+
+待探索：
+
+- 支持资源扩展名检测及使用 [rel=preload](https://w3c.github.io/preload/) 获取高优资源。
+- 使用 [Priority Hints](https://github.com/WICG/priority-hints) 进行重要性提示。
+
+## Polyfills
+
+`quicklink`:
+
+- [requestIdleCallback](https://developer.mozilla.org/en-US/docs/Web/API/Window/requestIdleCallback) 的一个非常小的回退。
+- 需要支持 `IntersectionObserver` （请参阅 [CanIUse](https://caniuse.com/#feat=intersectionobserver)）。我们推荐使用 Polyfill.io 等服务选择性地实现此功能：
+
+```html
+<script src="https://polyfill.io/v2/polyfill.min.js?features=IntersectionObserver"></script>
+```
+
+或者，请参见 [Intersection Observer polyfill](https://github.com/w3c/IntersectionObserver/tree/master/polyfill)。
+
+## 方法
+
+### 为预获取操作自定义超时时间
+
+默认超时时间为 2 秒（通过 `requestIdleCallback`），这里我们重写为 4 秒：
+
+```js
+quicklink({
+  timeout: 4000
+});
+```
+
+### 设置用于检测链接的 DOM 元素
+
+默认值为 `document`。
+
+```js
+const elem = document.getElementById('carousel');
+quicklink({
+  el: elem
+});
+```
+
+### 自定义预获取 URL 数组
+
+如果你想指定用于预获取的静态 URL 列表，而不是视区内的链接，你可以使用自定义 URL。
+
+```js
+quicklink({
+   urls: ['2.html','3.html', '4.js']
+});
+```
+
+### 为预获取设置请求优先级
+
+默认为低优先级（`rel=prefetch` 或 XHR）。对于高优先级（`priority: true`）的操作，尝试使用 `fetch()` 或退阶使用 XHR。
+
+```js
+quicklink({ priority: true });
+```
+
+### 自定义受允许请求源列表
+
+指定可被预获取的主机名列表，默认情况下仅允许同源主机名。
+
+> **划重点**：你还得加上自己的主机名！
+
+```js
+quicklink({
+  origins: [
+    // 添加我自己的
+    'my-website.com',
+    'api.my-website.com',
+    // 添加第三方的
+    'other-website.com',
+    'example.com',
+    // ...
+  ]
+});
+```
+
+### 允许所有源
+
+允许所有跨域请求。
+
+> **注意**：可能会导致 [CORB](https://chromium.googlesource.com/chromium/src/+/master/services/network/cross_origin_read_blocking_explainer.md) 以及 [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) 问题！
+
+```js
+quicklink({
+  origins: true,
+  // 或者
+  origins: []
+});
+```
+
+### 自定义忽略模式
+
+以下过滤器会在匹配 `origin` 之后执行。Ignores 在避免大型文件下载或 DOM 属性动态响应时十分有用！
+
+```js
+// 默认启用同源限制。
+//
+// 这个示例会忽略所有对以下模式的请求：
+//  - 所有 "/api/*" 路径名
+//  - 所有 ".zip" 扩展名
+//  - 所有带有 noprefetch 扩展名的 <a> 标签
+//
+quicklink({
+  ignores: [
+    /\/api\/?/,
+    uri => uri.includes('.zip'),
+    (uri, elem) => elem.hasAttribute('noprefetch')
+  ]
+});
+```
```

Comment:
Thanks for updating these to include `ignores` and `origins` from the latest release!